settings.gradle

```diff
@@ -1,7 +1,7 @@
 includeBuild ('../annotation-tools/annotation-file-utilities') {
     if (!file('../annotation-tools/annotation-file-utilities').exists()) {
         exec {
-            executable '.ci-build-without-test.sh'
+            executable './.ci-build-without-test.sh'
```

Comment:
Hi @zcai1 could you break out this change so that we can merge it?
It looks like https://github.com/opprop/value-inference/pull/8/files is failing for this reason, too.

Reply:
Ok, let me open a new PR

Reply:
Thanks for #387